db/migrations/versions/f586493c6c55_.py

```diff
@@ -0,0 +1,37 @@
+"""empty message
+
+Revision ID: f586493c6c55
+Revises: 3a22407701c8
+Create Date: 2023-01-18 10:38:25.817607
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from alembic_utils.pg_extension import PGExtension
+from sqlalchemy import text as sql_text
+
+# revision identifiers, used by Alembic.
+revision = 'f586493c6c55'
+down_revision = '3a22407701c8'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.get_context().autocommit_block():
+        op.execute("ALTER TYPE status ADD VALUE 'QA_COMPLETE'")
+
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.execute("ALTER TYPE status RENAME TO status_old")
+    op.execute("CREATE TYPE status AS ENUM('NOT_STARTED', 'IN_PROGRESS', 'SUBMITTED', 'COMPLETED')")
+    op.execute((
+        "ALTER TABLE transactions ALTER COLUMN status TYPE status USING "
```

Comment:
our table name is not `transactions`. Please can we use the correct table name?

Reply:
sorry missed that! have changed this now 👍🏾